readthedocs/core/resolver.py

```diff
@@ -345,6 +352,10 @@ def _use_subdomain(self):
         """Make decision about whether to use a subdomain to serve docs."""
         return settings.USE_SUBDOMAIN and settings.PUBLIC_DOMAIN is not None
 
+    def _use_cname(self, project):
+        """Test if to allow direct serving for project on CNAME."""
+        return True
```

Comment:
What's the case where this would be `False`?

Reply:
On .com this checks for the org subscription